### PR TITLE
Download block FED

### PR DESCRIPTION
**Pull request by liamjohnston**
Download(s) block.

I am absolutely loving how `:where()` means I don't have to come up with a bunch of BEM classnames 😍 

Screenshot 2024-05-14 at 11 24 48 AM

Screenshot 2024-05-14 at 11 25 09 AM

### CODE REVIEW

File: cdhweb/pages/templates/cdhpages/blocks/download_block.html [hunk 1]
```diff
@@ -9,11 +9,17 @@ <h2>{{ self.heading }}</h2>
     <p>{{ self.description }}</p>
   {% endif %}
 
-  <ul class="downloads__list">
+  <ul>
     {% for file in file_links %}
       <li>
-        <a {% if file.file.url %}href="{{ file.file.url }}"{% endif %} class="downloads__link" download>
-          {{ file.title }} ({{ file.file_size | filesizeformat }} {{ file.file_type }})
+        <a href="{{ file.file.url }}" download>
```
Comment (liamjohnston) [inline cdhweb/pages/templates/cdhpages/blocks/download_block.html:15]: FYI @sarahframe I removed the href conditional here, as I don't see how someone could add a download link without one. That OK?

File: cdhweb/static_src/images/sprites_src/global/arrow-right.svg
Comment (liamjohnston) [inline cdhweb/static_src/images/sprites_src/global/arrow-right.svg:1]: Removed the default icon set that comes with fed starterkit, CDH has special icons.

File: cdhweb/static_src/global/components/download.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      align-items: flex-start;
+      gap: 8px;
+    }
+  }
+
+  :where(svg) {
+    flex-shrink: 0;
+    aspect-ratio: 18 / 25;
+    width: 18px;
+    position: relative;
```
Comment (cameralibre) [inline cdhweb/static_src/global/components/download.scss:44]: what's the `relative` for?

**Review by cameralibre — APPROVED** [review]
nice! yeah the markup is definitely tidier without all the classes, using :where() is a great approach for this kind of small & simple component.

Re: the icons,are there no icon hover styles on this project? It's unusual to have hard-coded colors like that.

File: cdhweb/static_src/global/components/download.scss (hunk 2 above)
Comment (liamjohnston) [reply cdhweb/static_src/global/components/download.scss:44]: to make the next line (`top`) work

Comment (cameralibre) [reply cdhweb/static_src/global/components/download.scss:44]: ahhh

**Comment by liamjohnston** [comment]
>Re: the icons,are there no icon hover styles on this project? It's unusual to have hard-coded colors like that.

Yeah, good question. There are some hover styles, but I've not gotten to them yet. The only one I've seen in figma so far is the main nav (open vs closed state, actually):
Screenshot 2024-05-14 at 11 47 37 AM
Screenshot 2024-05-14 at 11 47 41 AM

So I think I'm going to have to go and update these icons so that the icon "shadow" colour is changeable. Usually we'd use currentColor as the icons would be one colour only. But in this case the main bit of the icon is always(?) black and it's the shadow bit that changes, but not to match the text.

**Comment by cameralibre** [comment]
yeah, you could use `currentColor` for the shadow (setting color on the icon rather than inheriting), or you could update a variable?

**Pull request merged**